setup.py

```diff
@@ -16,25 +16,28 @@
 
 setup(
     name="odoo-test-helper",
+    use_scm_version=True,
     description="Odoo Test Helper",
     long_description="\n".join(long_description),
     long_description_content_type="text/x-rst",
-    use_scm_version=True,
     packages=["odoo_test_helper"],
     include_package_data=True,
     setup_requires=["setuptools_scm"],
     license="AGPLv3+",
-    author="Akretion",
-    author_email="contact@akretion.com",
-    url="http://github.com/akretion/odoo-test-helper",
+    author="Odoo Community Association (OCA)",
+    author_email="support@odoo-community.org",
+    url="http://github.com/OCA/odoo-test-helper",
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: "
-        "GNU Affero General Public License v3 or later (AGPLv3+)",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
```

Comment:
Do we want to use LGPL here so we can use it in LGPL addons ?

Reply:
oh, good point. Fine for me. @sebastienbeau @lmignon I assume is fine for you both :wink:

Reply:
@sbidoul OTOH technically this is not a dependency to ship a module.
BTW we should get back to talk about adding `tests_require` key in the odoo manifest :wink:

Reply:
> BTW we should get back to talk about adding tests_require key in the odoo manifest

Yep. It should not be particularly difficult:

1. decide on the key - python-test in external_dependencies ?
2a. update oca/oca-ci to use `manifestoo list-external-dependencies` for that key like it already does for `deb` dependencies
2b. better: update setuptools-odoo to add a `test` extra with these and update oca/oca-ci to do `pip install -e ./setup/theaddon[test]`
3. organise the transition, teach, ...

Reply:
where should we move this discussion? `setuptools-odoo`? I'll open an RFC.